streaming/__init__.py

```diff
@@ -13,3 +16,11 @@
     'StreamingDataLoader', 'StreamingDataset', 'CSVWriter', 'JSONWriter', 'MDSWriter', 'TSVWriter',
     'XSVWriter', 'LocalDataset', 'vision', 'text'
 ]
+
+IS_MACOS = sys.platform == 'darwin'
+
+# Set the multiprocessing start method to `fork` for MAC OS since
+# streaming uses a FileLock for sharing the resources between ranks
+# and workers which is Unpickleable except method `fork`.
```

Comment:
need to fix comment -- this works because fork doesn't pickle

Reply:
Done in https://github.com/mosaicml/streaming/pull/101/commits/b3e9ee30820f97457712ce7e529e12464add9cbe.